wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt

```diff
@@ -266,13 +173,19 @@ internal constructor(
   }
 }
 
+suspend fun Anchor.sep12(authToken: AuthToken): Sep12 {
+  return this.customer(authToken)
+}
+
 fun Anchor.sep24(): Sep24 {
   return this.interactive()
 }
```

Comment:
Why do we encourage the use of `anchor.interactive()` vs `anchor.sep24()`? The latter seems to be more straightforward, and makes it easy to add more SEPs without defining some one-word alias to it.

Reply:
It's just how it's been historically. I can rename it to sep24 and keep current alias there

Reply:
That's a good point, I'm also in favor of using `sep24()`. Could we please also rename the `Interactive` class to `Sep24` instead? So it's clear that it's the class that interfaces with all Sep-24 related stuff - what do you think?

Reply:
Yep that's the idea 👍🏼